<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yeni İleti Ekranı</string>
  </property>
  <widget class="QLineEdit" name="ileti_basligi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ileti Başlığı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>İleti İçeriği</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paylaşılacak Paylaşım Grupları</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paylaşılacak Gruplar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paylaşım Durumu</string>
   </property>
  </widget>
  <widget class="QComboBox" name="paylasim_durumu">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>490</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Taslak Olarak Kaydet</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Paylaş</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="onayla">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>490</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>İletiyi Onayla</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="gruplar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>461</width>
     <height>111</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Seç</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Tüm Gruplar</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="paylasim_gruplari">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Seç</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Paylaşım Grupları</string>
    </property>
   </column>
  </widget>
  <widget class="QTextEdit" name="ileti_icerigi">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>461</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>onayla</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>ileti_onay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ileti_onay()</slot>
 </slots>
</ui>
